packages/components/src/ListBox/src/ListBox.tsx

```diff
@@ -166,7 +172,16 @@ function ListBox<T extends object>(props: ListBoxProps<T>, ref: ForwardedRef<HTM
             >
                 {renderChildren()}
                 {isLoading && Array.from({ length: 5 }).map((_, index) => {
-                    return <ListBoxItem key={`loadingListItem_${index.toString()}`} id={`loadingListItem_${index.toString()}`} isLoading={isLoading} size={size} textValue={stringFormatter.format("ListBoxItem.loadingTextValue")} />;
+                    return (
+                        <ListBoxItem
+                            key={`loadingListItem_${index.toString()}`}
+                            id={`loadingListItem_${index.toString()}`}
+                            isLoading={isLoading}
+                            size={size}
+                            textValue={stringFormatter.format("ListBoxItem.loadingTextValue")}
+                            {...listBoxItemProps}
```

Comment:
this one is a tricky one.. This list item is basically a skeleton. I don't think it makes sense to name is listBoxItemProps. We should probably rename it to like "loadingItemProps" or "skeletonItemProps"

Reply:
Renamed to loadingListBoxItemProps